README.md

```diff
@@ -163,6 +164,33 @@ let feedFromTwitter user pwd = taskSeq {
+| **Implementation**         | State machine (statically compiled)                                               | No state machine, continuation style                                 |
+| **Semantics**              | `seq`-like: on-demand                                                             | `seq`-like: on-demand                                                |
+| **Support `let!`**         | All `task`-like: `Async<'T>`, `ValueTask<'T>`, `Task<'T>` or any `GetAwaiter()`   | `Async<'T>` only                                                     |
+| **Support `do!`**          | `Async<unit>`, `Task<unit>` and `Task`, `ValueTask<unit>` and `ValueTask`         | `Async<unit>` only                                                   |
+| **Support `yield!`**       | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence                                  | `AsyncSeq`                                                           |
+| **Support `for`**          | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence                                  | `AsyncSeq` any sequence                                              |
+| **Behavior with `yield`**  | Zero allocations, no `Task` or even `ValueTask` created                           | Allocated, an F# `Async` wrapped in a singleton `AsyncSeq`           |
+| **Conversion to other**    | `TaskSeq.toAsyncSeq`                                                              | `AsyncSeq.toAsyncEnum`                                               |
+| **Conversion from other**  | Implicit (yield!) or `TaskSeq.ofAsyncSeq`                                         | `AsyncSeq.ofAsyncEnum`                                               |
+| **Recursion in `yield!`**  | No (requires F# support, upcoming)                                                | Yes                                                                  |
+| **Based on F# concept of** | `task`                                                                            | `async`                                                              |
+| **`MoveNextAsync`**        | `ValueTask<bool>`                                                                 | `Async<'T option>`                                                   |
+| **`Current` internals**    | `ValueOption<'T>`                                                                 | `Option<'T>`                                                         |
+| **Cancellation**           | Implicit token governing iteration                                                | Implicit token passed to each subtask                                |
+| **Performance**            | Very high, negligible allocations                                                 | Slower, more allocations, due to using `async`                       |
+| **Parallelism**            | Possible with ChildTask, support will follow                                      | Supported explicitly                                                 |
```

Comment:
```suggestion
| **Parallelism**            | Possible with ChildTask; support will follow                                      | Extensive explicit helpers provided                                                 |
```

Reply:
I'll take the semicolon, keep the original wording. People can always go to the docs to find out.